task/vsc7448/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "task-vsc7448"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+cfg-if = "0.1.10"
+userlib = {path = "../../sys/userlib", features = ["panic-messages"]}
+ringbuf = {path = "../../lib/ringbuf" }
+drv-spi-api = {path = "../../drv/spi-api"}
+drv-stm32h7-gpio-api = {path = "../../drv/stm32h7-gpio-api", default-features = false}
```

Comment:
FWIW this library _has no_ features, so, you don't need to turn them off.

Reply:
Okay!  This is wrong in other places, so I opened #418 to fix it elsewhere.